pkg/plugin/plugin.go

```diff
@@ -105,18 +106,18 @@ func (p *plugin) GenerateARM(ctx context.Context, cs *api.OpenShiftManagedCluste
 
 func (p *plugin) InitializeCluster(ctx context.Context, cs *api.OpenShiftManagedCluster) error {
 	log.Info("initializing cluster")
-	initializer := initialize.NewSimpleInitializer(p.entry)
+	initializer := initialize.NewSimpleInitializer(p.entry, p.pluginConfig)
```

Comment:
am wondering whether we should just be passing `p` in each of these things and exporting the entry and config fields

Reply:
yes, that would make it neater

Reply:
I can't do this because we start getting import cycles